include/MainApplication.h

```diff
@@ -50,9 +50,13 @@ class MainApplication : public QApplication
 	bool event(QEvent* event) override;
 #ifdef LMMS_BUILD_WIN32
 	bool winEventFilter(MSG* msg, long* result);
-	bool nativeEventFilter(const QByteArray& eventType, void* message,
-				long* result);
-#endif
+#if (QT_VERSION < QT_VERSION_CHECK(6,0,0))
+	using EventFilter = long;
```

Comment:
I quite liked the `FilterResult` type name you used in #7086. I think it's more accurate than `EventFilter`, since `result` is the result and not the filter itself.

Reply:
Changed to `FilterResult`